docker-compose.yml

```diff
@@ -267,7 +263,7 @@ services:
   flowetl_init:
     container_name: flowetl_init
     <<: *flowetl-common
-    entrypoint: /bin/bash -c "airflow db init && airflow users create --role Admin --username admin --email admin --firstname admin --lastname admin --password admin"
+    entrypoint: /bin/bash -c "airflow db init && airflow users create --role Admin --username admin --email admin --firstname admin --lastname admin --password ${FLOWETL_WEBSERVER_PASSWORD:?Must set FLOWETL_WEBSERVER_PASSWORD env var}"
```

Comment:
Tricky question, but where will this get substituted? Will compose do it, or will it happen inside the container in bash? Cos if compose you'll be able to see it by inspecting the container, right?

Also should this be `command` rather than entrypoint, or do we need to deliberately override the entrypoint for some reason?

Reply:
Compose subs at run - they're basically preprocessor directives here.
This does need to be `entrypoint` as `command` goes through the Airflow entrypoint by default - this then continues to do Airflow things, the service never finishes and the compose does not continue.
The container gets stopped on completion, so the only place that this should appear is in it's `~/,history`, and that can be skipped with a leading space before the command

Reply:
Really? `docker run apache/airflow db init` seems to terminate? Are we doing something in our entry point to break that behaviour?

Also, there's an `init.sh` in our image which does the db init and creates a user, no?

Reply:
As far as I can tell, we're not calling `init.sh` directly, but I think that it's responsibilities are better split up between the 'init' container, the webserver container and wherever the pool setup lives (I suspect the scheudler). Am investigating what's going on with `flowetl db init` now

Reply:
No, it's terminates fine now. Weird; entrypoint removed, replaced with `bash -c "blabla"`